ui/app/components/das/recommendation-card.js

```diff
@@ -137,6 +140,15 @@ export default class DasRecommendationCardComponent extends Component {
     );
   }
 
+  get copyButtonLink() {
+    const path = this.router.urlFor('optimize.summary', this.args.summary.slug, {
+      queryParams: { namespace: this.args.summary.jobNamespace },
+    });
+    const { origin } = window.location;
+
+    return `${origin}${path}`;
+  }
```

Comment:
Kinda surprising that the router service doesn't have anything to get the full URL.

Reply:
I agree!!! I don’t like this 😐